csrc/scheduler/matmul_utils.cpp

```diff
@@ -424,7 +427,9 @@ std::string checkMatmulType(Fusion* fusion, const MmaOp* mma_expr) {
       if (val->definition()->isA<BroadcastOp>()) {
         const auto& bcast_inputs = val->definition()->inputs();
         // BroadcastOp has single input/output, not need to check other things
-        return bcast_inputs.front()->isFusionInput();
+        return bcast_inputs.front()->isFusionInput() ||
+            (dynamic_cast<LoadStoreOp*>(bcast_inputs.front()->definition()) !=
+             nullptr);
```

Comment:
In the future, we should not hard code this `fusion input -> transpose->broadcast -> mma input` order. We should allow fusing any order, such as `fusion input -> broadcast->transpose -> mma input`, etc.